include/envoy/event/dispatcher.h

```diff
@@ -148,11 +148,12 @@ class Dispatcher {
    * @param socket supplies the socket to listen on.
    * @param cb supplies the callbacks to invoke for listener events.
    * @param bind_to_port controls whether the listener binds to a transport port or not.
+   * @param backlog_size controls listener pending connections backlog
    * @return Network::ListenerPtr a new listener that is owned by the caller.
    */
   virtual Network::ListenerPtr createListener(Network::SocketSharedPtr&& socket,
-                                              Network::ListenerCallbacks& cb,
-                                              bool bind_to_port) PURE;
+                                              Network::ListenerCallbacks& cb, bool bind_to_port,
+                                              uint32_t backlog_size = ENVOY_TCP_BACKLOG_SIZE) PURE;
```

Comment:
Can we not have  a default param on this interface? I would prefer that we make sure we hit all call sites. How many other places need to change?

Reply:
It's definitely possible because the first version of the patch didn't use a default param :-). It's a decent amount of tests, but if you're okay with that, I can push an update.

Reply:
Yes please.

Reply:
Done!